### PR TITLE
test: Add an LLM-powered bot to detect dupes

### PR DESCRIPTION
**Pull request by zadjii-msft**
Just like in https://github.com/microsoft/WSL/pull/10745

We're working with the WSL team to figure out if we can use a LLM to help us triage. This _should_ just comment on issues, if it finds something similar on the backlog. 

@craigloewen-msft as an FYI. If this explodes or doesn't work, well, we'll figure it out soon enough 😝

### CODE REVIEW

**Comment by craigloewen-msft** [comment]
This won't work right away. Please message me on Teams and I can help onboard it <3

**Comment by zadjii-msft** [comment]
(Craig onboarded us)

File: .github/workflows/similarIssues.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  issues:
+    types: [opened]
+
+jobs:
+  getsimilarissues:
+    runs-on: ubuntu-latest
+    outputs:
+      message: ${{ steps.getbody.outputs.message }}
+    steps:
+      - uses: actions/checkout@v2
```
Comment (lhecker) [inline .github/workflows/similarIssues.yml:13]: Does this script really need to checkout something?

Comment (zadjii-msft) [reply .github/workflows/similarIssues.yml:13]: @craigloewen-msft cause I have no idea

Comment (craigloewen-msft) [reply .github/workflows/similarIssues.yml:13]: I believe it does to get the `app.js` file that runs for this action. Is it a problem to do the check out? If so I mean you can try and remove it and see what happens?

Comment (DHowett) [reply .github/workflows/similarIssues.yml:13]: `app.js` isn't part of our repo though. We shouldn't need to check out _our_ repo :gri:

Comment (craigloewen-msft) [reply .github/workflows/similarIssues.yml:13]: Ah user error when creating this repo. That line isn't needed! :)
```suggestion
```

**Review by lhecker — APPROVED** [review]

**Review by craigloewen-msft — APPROVED** [review]

**Comment by craigloewen-msft** [comment]
Needs 1 more approval for merge: @DHowett  ? :)

**Review by DHowett — APPROVED** [review]

**Pull request merged**